css-color-4/Overview.bs

```diff
@@ -2759,11 +2809,11 @@ Default Style Rules</h2>
 
 </pre>
 
-<h2 id="system-colors" class="no-num">
+<h2 id="deprecated-system-colors" class="no-num">
 Appendix A: Deprecated CSS System Colors</h2>
 
 	Earlier versions of CSS defined several additional named color keywords,
-	the <dfn>&lt;deprecated-system-color></dfn>s,
+	the <dfn id="deprecated-system-color" lt="deprecated system colors" export>system colors</dfn>&lt;deprecated-system-color></dfn>s,
```

Comment:
This change seems a bit strange; it's not clear why the `<deprecated-system-color>` text remains after the `dfn` element.  It might be OK as it was...

Reply:
This looks like a weird paste error I didn't catch, thanks @dbaron!